soc/st/stm32/stm32u0x/power.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright The Zephyr Project Contributors
```

Comment:
FYI (not blocking): This isn't wrong, but note that you could also use your own copyright.